### ISSUE TITLE
Jazzy can not parse open classes from Swift 3

### ISSUE THREAD

**Issue by rafiki270** [realm/jazzy#646]
```
import UIKit

/**
 Main class to interface with LiveUI system
 */
open class LUI: LUIMain, LUIProtocol {


    // MARK: Accessing data

    /**
     Get image by it's key, setup in the LiveUI admin panel

     - Parameters:
     - imageKey: Image key from LiveUI admin panel
     - defaultImage: Default image that should be used if image doesn't exist. Default is nil to be returned

     - Returns: UIImage Image or nil
     */
    open static func image(_ imageKey: String, defaultImage: UIImage? = nil) -> UIImage? {
        return Images.sharedInstance.get(imageKey, defaultImage: defaultImage)
    }
``
```

**Comment by picciano** [comment]
I'm confused on the status of this. I see the merge above, but doesn't look like Jazzy version has been updated and am not getting an update through gem. This is still failing with Swift 3/Xcode 8, is there something I need to do to get this to work locally?

**Comment by rafiki270** [comment]
I would get we'll have to wait for the next release if you want to use gem ...

**Comment by rafiki270** [comment]
also, the next big issue might be the underscore before a method variable or closure

**Comment by picciano** [comment]
Guess we'll find out soon enough. Xcode 8 GM should be announced in about 15 minutes or so...

**Comment by istx25** [comment]
Fixed as per #647.
